blueocean-rest/src/main/java/io/jenkins/blueocean/rest/OmniSearch.java

```diff
@@ -15,9 +21,21 @@
 
     public abstract String getType();
 
-    public abstract Pageable<T> search(Query q);
+    public abstract Pageable<T> search(Query q, ItemGroup<?> root);
```

Comment:
@kzantow No reference to hudson.model.* from blueocean-rest 🐛  Maybe some other abstract to define a context under which search needs to happen, maybe a Container?